test/integration/cart_integration_test.rb

```diff
@@ -64,35 +69,35 @@ class CartIntegrationTest < ActionDispatch::IntegrationTest
     click_link_or_button('Cart')
 
     within('#cart_items') do
-      assert page.has_content?('coffee')
+      assert page.has_content?('produce only the finest')
       assert page.has_content?('11')
     end
   end
 
   test 'an unauthorized user can add different items to the cart and show the correct price' do
-    coffee = Item.create(title: 'coffee', description: 'black nectar of the gods', price: 1200)
-    aeropress = Item.create(title: 'aeropress', description: 'light coffee', price: 1300)
-    visit "/items/#{coffee.id}"
+    bad_item = create(:item)
+    bad_item2 = create(:item)
+    visit "/items/#{bad_item.id}"
 
     select "1", from: 'cart[quantity]'
     click_link_or_button('Add to Cart')
-    visit "/items/#{aeropress.id}"
+    visit "/items/#{bad_item2.id}"
     select "2", from: 'cart[quantity]'
     click_link_or_button('Add to Cart')
     click_link_or_button('Cart')
 
     within('#cart_items') do
-      assert page.has_content?('coffee')
+      assert page.has_content?('espresso')
       assert page.has_content?('1')
-      assert page.has_content?('$12.00')
-      assert page.has_content?('aeropress')
+      assert page.has_content?('$8.00')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.